src/components/Comments/CommentBody.tsx

```diff
@@ -17,12 +17,12 @@ import AnnotationReply from '@manuscripts/assets/react/AnnotationReply'
 import { Keyword, UserProfile } from '@manuscripts/json-schema'
 import { Field, FieldProps, Form, Formik } from 'formik'
 import React, { Dispatch, SetStateAction, useEffect } from 'react'
-import ReactTooltip from 'react-tooltip'
 import styled from 'styled-components'
 
 import { CommentType, UnsavedComment } from '../../lib/comments'
 import { ButtonGroup, PrimaryButton, SecondaryButton } from '../Button'
 import { FormError } from '../Form'
+import { Tooltip } from '../Tooltip'
```

Comment:
As I mentioned on article editor. Please separate cleanup into a different PR. Thanks

Reply:
I don't think this is the same. Whereas in article-editor, I can see how someone can get confused by why some of the changes are there, I think these changes fit this PR; there are no business logic changes.